apps/web/src/pages/Applications/ApplicationProfilePage/components/PersonalInformation/Display.tsx

```diff
@@ -0,0 +1,167 @@
+import React from "react";
+import { useIntl } from "react-intl";
+
+import { User } from "@gc-digital-talent/graphql";
+
+import { getFullNameHtml } from "~/utils/nameUtils";
+import {
+  commonMessages,
+  getArmedForcesStatusesProfile,
+  getCitizenshipStatusesProfile,
+  getLanguage,
+  getProvinceOrTerritory,
+} from "@gc-digital-talent/i18n";
+
+import { empty } from "@gc-digital-talent/helpers";
+import FieldDisplay from "../FieldDisplay";
+
+interface DisplayProps {
+  user: User;
+}
+
+const Display = ({
+  user: {
+    firstName,
+    lastName,
+    email,
+    telephone,
+    preferredLang,
+    preferredLanguageForInterview,
+    preferredLanguageForExam,
+    currentCity,
+    currentProvince,
+    citizenship,
+    armedForcesStatus,
+  },
+}: DisplayProps) => {
+  const intl = useIntl();
+  const notProvided = intl.formatMessage(commonMessages.notProvided);
+
+  return (
```

Comment:
I'm going to try and sync these up better but first, are you referring to the copy, layout/order or both? 

The layout is slightly different to consider reading order and ATs. Not to say the order cannot match exactly and keep proper reading order, it just may be tricky since the text flows from left to right so we would need to create columns to read it more like a newspaper. Honestly, this was me just going for the easy path 😅 

For the copy, the design omitted some of the fields and I wasn't quite sure why so I did the best I could with the data we currently show vs what is in the design itself 😕 Some copy was slightly tweaked to provide more context for AT's as well.

Reply:
Both copy and layout.  I don't have a strong opinion about this, honestly.  I think it's fine but I noticed it because it's so different from the design.  Might want to check in with Josh on this, too.

Reply:
Alright, I have update this to have the copy and layout match the design. 

[update layout to match design](https://github.com/GCTC-NTGC/gc-digital-talent/pull/6296/commits/154db89df48bd13b353b0e242d114003604845a9)

Reply:
Looks reasonably close now.  :+1: